python/CHANGELOG.rst

```diff
@@ -5,7 +5,8 @@
 **Fixes**
 
  - ``ts.subset()`` produces valid tree sequences even if nodes are shuffled
-   out of time order (:user:`hyanwong`, :pr:`2479`, :issue:`2473`)
+   out of time order (:user:`hyanwong`, :pr:`2479`, :issue:`2473`), and the
+   same for ``tables.subset()`` (:user:`hyanwong`, :pr:`2489`)
```

Comment:
Is there something about the output of subset we need to note, since we're doing an extra sort?

Reply:
Yes, I think so.

Reply:
OK, done.